components/Dialogs/src/MDCDialogPresentationController.h

```diff
@@ -42,6 +42,15 @@
  */
 @property(nonatomic, assign) BOOL dismissOnBackgroundTap;
 
+/**
+ Customize the corner radius of the shadow to match the presented view's corner radius.
+ If the presented view corner radius and presentedViewCornerRadius are different, the rendered
+ shadow will not match.
+ 
+ Defaults to 0.0.
+ */
+@property(nonatomic, assign) CGFloat presentedViewCornerRadius;
```

Comment:
Ahh so I've filed https://github.com/material-components/material-components-ios/issues/4263 to think a bit further ahead here, but I think we need to call this API `dialogCornerRadius`. "Presented view" is still an implementation detail and the current documentation doesn't feel like a desired final implementation.

Instead, I'd hope this API's documentation would eventually be:

```
The corner radius of the presented dialog.

Defaults to 0.
```

And then the implementation would simply set the cornerRadius on both the tracking view and the presented view for now. Once https://github.com/material-components/material-components-ios/issues/4263 is resolved though this would only have to set the cornerRadius on the container view.

Reply:
Done.